src/components/KichlerPage/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { css } from '@emotion/react';
+import NextImage from 'next/image';
+import type { FC } from 'react';
+
+import {
+  CaseStudyBottomNav,
+  CaseStudyHeaderPane,
+  CaseStudyIntroContentPane,
+  CaseStudyResultsPane,
+  CaseStudySummaryPane,
+  Layout,
+  TrademarkSymbol,
+} from '~/components';
+import { atMinDesktop, spacing } from '~/theme';
+
+import { kichlerNav } from '../CaseStudy/navProps';
+
+export const KichlerPage: FC = () => {
+  const CaseStudyResultsProps = {
+    children: [
+      {
+        heading: '97/100',
+        text: 'Average desktop performance score, increased by 41%',
+      },
+      {
+        heading: 'Compliant',
+        text: 'Data privacy requirements',
+      },
+      {
+        heading: '868',
+        text: 'Overall health score, increased by 24%',
+      },
+      {
+        heading: '100/100',
+        text: 'A11Y score',
+      },
```

Comment:
```suggestion
      },
      {
      heading: 'Partner',
      text: 'Bonfire Red'
      },
```

can we add a link to their site too?    
Not sure if it makes sense here, but we should call out the partnership sooner. 
```